definitions/infra-awsalbtargetgroup/golden_metrics.yml

```diff
@@ -28,6 +30,7 @@ unhealthyHosts:
       eventName: entityName
 responseTime:
   title: Average response time
+  unit: SECONDS
   queries:
     aws:
       select: average(aws.applicationelb.TargetResponseTime)
```

Comment:
double checking: is this metric actually on seconds or we need to convert it to seconds?